torch/csrc/jit/pybind_utils.h

```diff
@@ -693,6 +710,13 @@ inline py::object toPyObject(IValue ivalue) {
     AT_ASSERT(classType);
     auto pyClass =
         py::module::import("torch.jit").attr("_get_script_class")(obj->name());
```

Comment:
given tat it's in the hot path - shall we import torch.jit and lookup the method only once somewhere above? (not sure whether it matters)